Sources/TuistKit/Mappers/Graph/StaticXCFrameworkModuleMapGraphMapper.swift

```diff
@@ -0,0 +1,182 @@
+import Foundation
+import Path
+import TuistCore
+import TuistSupport
+import XcodeGraph
+
+public final class StaticXCFrameworkModuleMapGraphMapper: GraphMapping {
+    private let fileHandler: FileHandling
```

Comment:
Can't we use `FileSystem` here?

Reply:
It'd mean rewriting the `GraphMapping` module which is out of scope here